assets/dev/js/editor/elements/views/behaviors/sortable.js

```diff
@@ -62,6 +112,12 @@ SortableBehavior = Marionette.Behavior.extend( {
 			},
 			sortableOptions = _.extend( defaultSortableOptions, this.view.getSortableOptions() );
 
+		// Add a swappable behavior ( used for flex containers ).
```

Comment:
```suggestion
		// Add a swappable behavior (used for flex containers).
```

Reply:
Done. 
(Even though I always do that... 🙄)